### PR TITLE
Windows + Frames can be attacked in help intent

### PR DESCRIPTION
**Pull request by Xander3359**
## About The Pull Request
Windows + Frames can be attacked in help intent
## Why It's Good For The Game
QOL
## Changelog
:cl:
qol: Windows + Frames can be attacked in help intent
/:cl:

### CODE REVIEW

**Comment by XElectricX** [comment]
This sounds awful. And from looking at the code, it doesn't even do what it says on the tin? It looks like attack procs will only execute if _not_ on harm intent.

File: code/game/objects/structures/window.dm [hunk 1]
```diff
@@ -184,12 +184,19 @@
...
 
-	else if(iscrowbar(I) && reinf && state <= 1 && deconstructable)
+	else if(iscrowbar(attacking_item) && reinf && state <= 1 && deconstructable)
 		dismantle = TRUE
 		state = 1 - state
 		playsound(loc, 'sound/items/crowbar.ogg', 25, 1)
 		to_chat(user, (state ? span_notice("You have pried the window into the frame.") : span_notice("You have pried the window out of the frame.")))
 
+	if(user.a_intent == INTENT_HARM)
+		return
```
Comment (XElectricX) [inline code/game/objects/structures/window.dm:194]: ???

Comment (Xander3359) [reply code/game/objects/structures/window.dm:194]: Harm intent is handled on the parent call.

**Comment by TiviPlus** [comment]
I dont get it whats the issue this is fixing

**Comment by Lumipharon** [comment]
I unno how this is QOL, it just makes it behave inconsistantly?

**Pull request closed without merging**